nixos/modules/services/monitoring/prometheus/default.nix

```diff
@@ -738,11 +734,23 @@ in {
         '';
       };
 
-      alertmanagerURL = mkOption {
-        type = types.listOf types.str;
+      alertmanagers = mkOption {
+        type = types.listOf types.attrs;
+        example = literalExample ''
+          [ {
```

Comment:
(nitpick) Isn't it sufficient to write the expression directly without `literalExample`? (Definetely not a blocker for this, just wondering as I did this in the past and it worked perfectly fine :) )

Reply:
* Currently the option looks like this in the rendered manual:
![literal-example](https://user-images.githubusercontent.com/20464732/62853296-2f194600-bcec-11e9-9e3c-a390d15109e6.png)

* leaving away `literalExample` but keeping the single quotes would result in this wrong example:
![string-only](https://user-images.githubusercontent.com/20464732/62853364-67b91f80-bcec-11e9-9519-a51f3dcfe408.png)

* And also leaving away the single quotes results in this:
![plain](https://user-images.githubusercontent.com/20464732/62853607-33922e80-bced-11e9-8e0c-d89fe4a5d87a.png)

The last one made me realize, that I forgot a semicolon in the example, which I'll fix in a moment :)
I'd still go with the current version.

Reply:
I see... I was pretty sure that `example = { /* ... */ }` also gets pretty-printing, but obviously I was wrong :sweat_smile: